controller/search.go

```diff
@@ -243,3 +247,47 @@ func (c *SearchController) Users(ctx *app.UsersSearchContext) error {
 
 	return ctx.OK(&response)
 }
+
+// Iterate over the WI list and read parent IDs
+// Fetch and load Parent WI in the included list
+func (c *SearchController) enrichWorkItemList(ctx *app.ShowSearchContext, res *app.SearchWorkItemList) {
+	// Need a map of string to UUID to prevent duplicates in place.
+	visitedIDs := map[string]uuid.UUID{}
+	fetchInBatch := []uuid.UUID{}
+	for _, wi := range res.Data {
+		cond := wi.Relationships != nil && wi.Relationships.Parent != nil && wi.Relationships.Parent.Data != nil
```

Comment:
you can write it in a single line: 
`if wi.Relationships != nil && wi.Relationships.Parent != nil && wi.Relationships.Parent.Data != nil {`

Reply:
Done in https://github.com/fabric8-services/fabric8-wit/pull/1592/commits/e7111344d1c6c8e78055c45e2e53b217e1f40696